sdk/monitor/Azure.Monitor.Query/src/Models/MetricValue.cs

```diff
@@ -9,6 +10,23 @@ public partial class MetricValue
     {
         private string _toString;
 
+        /// <summary> Initializes a new instance of MetricValue. </summary>
+        /// <param name="timeStamp"> the timestamp for the metric value in ISO 8601 format. </param>
+        /// <param name="average"> the average value in the time range. </param>
+        /// <param name="minimum"> the least value in the time range. </param>
+        /// <param name="maximum"> the greatest value in the time range. </param>
+        /// <param name="total"> the sum of all of the values in the time range. </param>
+        /// <param name="count"> the number of samples in the time range. Can be used to determine the number of values that contributed to the average value. </param>
+        public MetricValue(DateTimeOffset timeStamp, double? average, double? minimum, double? maximum, double? total, double? count)
```

Comment:
Why make the ctor public VS adding a factory method like we did for everything else?

Reply:
updated